job/server/pom.xml

```diff
@@ -30,6 +30,11 @@
 
   <dependencies>
     <!-- External dependencies -->
+    <dependency>
+      <groupId>com.github.oshi</groupId>
+      <artifactId>oshi-core</artifactId>
+      <version>${oshi.version}</version>
+    </dependency>
```

Comment:
Assuming you went through the necessary hoops dealing with the license on this library?